homeassistant/components/flipr/coordinator.py

```diff
@@ -6,39 +6,44 @@
 from flipr_api import FliprAPIRestClient
 from flipr_api.exceptions import FliprError
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
-from .const import CONF_FLIPR_ID
-
 _LOGGER = logging.getLogger(__name__)
 
 
-class FliprDataUpdateCoordinator(DataUpdateCoordinator):
-    """Class to hold Flipr data retrieval."""
+class BaseDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
Why do we make this a base one? I don't see a direct need for that

Reply:
This is a preparation for the Hub device (next PR with a switch platform). The Base is common behaviour to Hub and Flipr.

Reply:
Let's keep this PR scoped at moving to 1 integration per account

Reply:
ok, kept only FliprDataUpdateCoordinator